### PR TITLE
Refacto gruntfile

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra/pull/492
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/799

### CODE REVIEW

File: grunt_tasks/coffee_discovering_task.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module.exports = function(grunt) {
```
Comment (nicolasThal) [inline grunt_tasks/coffee_discovering_task.js]: wouldn't it be possible to move those files in the BackofficeBundle also?

**Comment by itkg-nanne** [comment]
:+1:

File: grunt_tasks/coffee_discovering_task.js (hunk 1 above)
Comment (itkg-ngilain) [inline grunt_tasks/coffee_discovering_task.js]: It's an open question : for now I moved only what was strongly related to open-orchestra. The remaining tasks are generics so I leave there where they are.
But another way to think is effectively to move all tasks to open-orchestra-cms-bundle. What is your preference ?

Comment (nicolasThal) [inline grunt_tasks/coffee_discovering_task.js]: I would rather move the remaining task about coffee  and less in the cms-bundle, they are really needed for open-orchestra I think

Comment (itkg-ngilain) [inline grunt_tasks/coffee_discovering_task.js]: ok

**Comment by itkg-ngilain** [comment]
@nicolasThal : I moved coffee & less discovering tasks. Is it ok for you ?

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**